docs/www/tls-kubernetes.md

```diff
@@ -0,0 +1,375 @@
+
+We can then enable client authentication (mTLS) on the same listener by setting `requireClientAuth` to `true`.
+
+So for example, we can edit our previous `yaml` file to add this new setting: 
+
+```yaml
+kafkaApi:
+    - port: 9092
+    tls:
+        enabled: true
+        requireClientAuth: true
+```
+
+When you enable client authentication, you must specify a key-pair (certificate and key). As we will later see, the operator can also generate the key-pair.
+
+When you finish setting up your configuration, apply it by running:
+
+```bash
+kubectl apply \
+-f <your-file-path.yaml>
+```
+
+Remember to change to the actual file path that you used. 
+
+
+## Managing Certificates
+
+### Certificates Creation
+
+The Redpanda operator uses [cert-manager](https://cert-manager.io) to generate certificates. 
+
+By default, when you enable TLS, the Redpanda operator generates a root certificate for each API and uses that to generate leaf certificates for the nodes and client. You can check what certificates were generated in the section down below. 
+
+Specifically for the Kafka API, the operator also offers two options: 
+
+1) provide an existing certificate issuer; 
+   
+2) provide a node certificate. 
+
+This functionality is only available for the Kafka API.
+
+To provide an existing certificate issuer we need to fill in the `issuerRef`.
+
+In the `yaml` configuration file add this section:
+
+```yaml
+    kafkaApi:
+    - port: 9092
+      tls:
+        enabled: true
+        issuerRef:
+          name: <issuer-name>
+          kind: ClusterIssuer
+```
+
+Similarly, to provide a particular certificate (as a secret) as the node certificate we need to fill in the `nodeSecretRef` field:
+
+```yaml
+    kafkaApi:
+    - port: 9092
+      tls:
+        enabled: true
+        nodeSecretRef:
+          name: cluster-tls-node-certificate
+          namespace: given-cert
+```
+
+If the secret is in a namespace different from the namespace of the Redpanda cluster, the operator copies it over.
+
+### Reading/Inspecting the certificates
+
+When enabling TLS, the Redpanda operator generates a root certificate for the Kafka API that is consecutively used to generate a “node” certificate for the Kafka API listener. 
+
+If client authentication (mTLS) is enabled, an additional “client” certificate is generated. Each certificate is stored in a Kubernetes secret resource under the same namespace as the Redpanda cluster. The Certificate name is the same as that of the corresponding secret.
+
+Here’s the list of secrets for the Kafka API:
+
+Root certificate
+* Used to generate the node and client certificates
+* Secret: `<cluster>-kafka-root-certificate`
+
+Node certificate
+* The node certificate is automatically provided to Redpanda by the operator. In particular, the certificate secret is mounted as a volume that is consumed by Redpanda.
+* Secret: `<cluster>-redpanda`
+
+Client certificate (applicable only when client authentication is enabled)
```

Comment:
```suggestion
Client certificate (applicable only when mTLS is enabled)
```
Client certification and mTLS are the same thing here, right? Can we make sure we're consistent in referring to it as one or the other?

Reply:
Yes. For this context, same thing.